File: 백준/Silver/1769. 3의 배수/3의 배수.py
n = input()
cnt = 0

while len(n) > 1:
    sum = 0
    for i in range(len(n)):
        sum += int(n[i])
    n = str(sum)
    cnt += 1
print(cnt)
if int(n) % 3 == 0:
    print("YES")
else:
    print("NO")

File: 백준/Silver/1769. 3의 배수/3의 배수.kt
fun main() {
    var n = readLine()!!
    var cnt = 0

    while (n.length > 1) {
        var summ = 0
        for (i in 0 until n.length) {
            summ += n[i].toString().toInt()
        }
        n = summ.toString()
        cnt ++
    }
    println(cnt)

    if (n.toInt() % 3 == 0){
        println("YES")
    } else {
        print("NO")
    }
}